ui/components/Filters.js

```diff
@@ -79,53 +74,52 @@ import {
 import LoadingScreen from './LoadingComponents/LoadingComponent';
 import { useGetProviderCapabilitiesQuery } from '@/rtk-query/user';
 
-const styles = (theme) => ({
-  grid: {
-    padding: theme.spacing(2),
-  },
-  tableHeader: {
-    fontWeight: 'bolder',
-    fontSize: 18,
-  },
-  createButton: {
-    width: 'fit-content',
-    alignSelf: 'flex-start',
-  },
-  viewSwitchButton: {
-    justifySelf: 'flex-end',
-    paddingLeft: '1rem',
-  },
-  ymlDialogTitle: {
-    display: 'flex',
-    alignItems: 'center',
-  },
-  searchWrapper: {
-    justifySelf: 'flex-end',
-    marginLeft: 'auto',
-    paddingLeft: '1rem',
-    display: 'flex',
-  },
-
-  ymlDialogTitleText: {
-    flexGrow: 1,
-  },
-  fullScreenCodeMirror: {
+const GridWrapper = styled('div')(({ theme }) => ({
```

Comment:
move these repeatative styles in filters and patterns to common styles folder please so this files will be less cluttered with styles and logic